Expensify/App
# Upgraded react-native-modal to v13.0.0

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/7293

### Tests


1. Start a chat with a user.
2. Click on your avatar in the chat and verify that no Message button is visible.
3. Now click on another user's avatar and verify that the Message button is visible and clicking it opens the chat with that user.
4. Check the JS console
Check the JS console


### QA Steps


1. Start a chat with a user.
2. Click on your avatar in the chat and verify that no Message button is visible.
3. Now click on another user's avatar and verify that the Message button is visible and clicking it opens the chat with that user.
4. Check the JS console


### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/48352577/150340490-aed4f28c-0add-4539-b204-c1881f3bce79.mp4


#### Mobile Web


#### Desktop


#### iOS


#### Android
